projects/plugins/jetpack/functions.compat.php

```diff
@@ -54,14 +54,14 @@ function jetpack_get_youtube_id( $url ) {
 	return $id;
 }
 
-if ( ! function_exists( 'youtube_sanitize_url' ) ) :
+if ( ! function_exists( 'jetpack_youtube_sanitize_url' ) ) :
```

Comment:
Seems odd that this function is defined both here and in modules/shortcodes/youtube.php. Looks like that goes way back, 51fe166a added one copy in 2012 and 661bd1b1 another in 2013.

I note this copy has the wrong return type (should be `string|false`), while the other still has a comment with `http://youtube.com` rather than `https://youtube.com` and one with "Youtube" rather than "YouTube". Could be worse, they could be even more different.

Reply:
Yeah, I figured perhaps there are scenarios that the function needs to be present earlier than the modules load. Did some clean up in ac10a4aab4.

Reply:
I wonder if the other copy ever gets used, since this file seems like it's always loaded first. Maybe on Simple, but then maybe there it should be defined in `wp-content/mu-plugins/shortcodes/youtube-wpcom.php`?

Anyway, that can be a matter for a different PR.